packages/dds/tree/src/feature-libraries/sequence-field/compose.ts

```diff
@@ -40,24 +41,40 @@ export type NodeChangeComposer<TNodeChange> = (changes: TNodeChange[]) => TNodeC
  * - Support for slices is not implemented.
  */
 export function compose<TNodeChange>(
-    changes: Changeset<TNodeChange>[],
+    changes: TaggedChange<Changeset<TNodeChange>>[],
     composeChild: NodeChangeComposer<TNodeChange>,
 ): Changeset<TNodeChange> {
+    if (changes.length === 0) {
+        return [];
+    }
+
     if (changes.length === 1) {
-        return changes[0];
+        return changes[0].change;
     }
-    let composed: Changeset<TNodeChange> = [];
-    for (const change of changes) {
-        composed = composeMarkLists(composed, change, composeChild);
+    let composed: Changeset<TNodeChange> = clone(changes[0].change);
```

Comment:
Doen't this usage of `clone` violate the rule that "Every child included in the composed change must be the result of a call to `composeChild`"?

Reply:
I don't think so, since the cloned changes will not be directly included in the return value. I think I will rewrite this section to be easier to understand.